gobblin-service/src/test/java/org/apache/gobblin/service/monitoring/MysqlJobStatusRetrieverTest.java

```diff
@@ -97,31 +97,31 @@ public void testGetFlowStatusFromJobStatuses() throws Exception {
 
     addJobStatusToStateStore(flowExecutionId, JobStatusRetriever.NA_KEY, ExecutionStatus.COMPILED.name());
     Assert.assertEquals(ExecutionStatus.COMPILED,
-        jobStatusRetriever.getFlowStatusFromJobStatuses(jobStatusRetriever.getJobStatusesForFlowExecution(FLOW_NAME, FLOW_GROUP, flowExecutionId)));
+        JobStatusRetriever.getFlowStatusFromJobStatuses(jobStatusRetriever.getJobStatusesForFlowExecution(FLOW_NAME, FLOW_GROUP, flowExecutionId)));
```

Comment:
again, let's reorder only if we go ALL-IN w/ this `getJobStatusesForFlowExecution` too